source/deployment/sso-saml-adfs.rst

```diff
@@ -91,6 +91,9 @@ Create Claim Rules
 
 For Mattermost 3.4 and later, the *FirstName* and *LastName* attributes are optional.
 
+.. note::
+It is recommended that you configure a claim rule that associates a unique, unchanging identifier with the **Outgoing Claim Type** of ``IdAttribute``. Please check with your ADFS admin to determine which LDAP Attribute you should use. `More information is available here. <#how-to-bind-authentication-to-id-attribute-instead-of-email>`__
```

Comment:
```suggestion
  It is recommended that you configure a claim rule that associates a unique, unchanging identifier with the **Outgoing Claim Type** of ``IdAttribute``. Please check with your ADFS admin to determine which LDAP Attribute you should use. `More information is available here. <#how-to-bind-authentication-to-id-attribute-instead-of-email>`__
```

Reply:
Sorry there are still formatting issues and some of my previous suggestions got lost somehow.